region/norfair/east.json

```diff
@@ -3161,9 +3157,30 @@
           "to": [
             {
               "id": 2,
-              "requires": null,
-              "unlock": null,
-              "note": "Some lava damage is required without Morph. Is it worth writing down?"
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"or": [
+                  "Morph",
+                  "canSwim",
+                  "canSuitlessLavaDive"
```

Comment:
I didn't put `canSuitlessLavaDive` in for this room because it's pretty shallow and I didn't think you needed to get into the lava too much. You feel this is enough to require the tech?

Reply:
Can't say I have a strong opinion about it either way. The amount of lava frames needed to traverse without it might be an indication of what to do with it but it was certainly a relevant amount of damage to have in logic.